core/services/list_models.go

```diff
@@ -0,0 +1,72 @@
+package services
+
+import (
+	"regexp"
+
+	"github.com/go-skynet/LocalAI/core/config"
+	"github.com/go-skynet/LocalAI/core/schema"
+	"github.com/go-skynet/LocalAI/pkg/model"
+)
+
+type ListModelsService struct {
+	bcl       *config.BackendConfigLoader
+	ml        *model.ModelLoader
+	appConfig *config.ApplicationConfig
+}
+
+func NewListModelsService(ml *model.ModelLoader, bcl *config.BackendConfigLoader, appConfig *config.ApplicationConfig) *ListModelsService {

```

Comment:
as we are refactoring, I wonder if this shouldn't just be a handler for the application, or just a function instead? It does not seem to be a long-living service, having it inside services makes me wonder if this is something that lives e.g. in a go routine

Reply:
I'd like to address any design concerns here in the next PR unless you consider it a blocker - wiring up the services will be the focus of that one